resqpy/well/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""resqpy modules providing trajectory, deviation survey, blocked well, wellbore frame, wellbore marker frame and
+ md datum classes.
+"""
+
+__all__ = [
+    'md_datum', 'blocked_well', 'well_object_funcs', 'trajectory', 'deviation_survey', 'wellbore_frame',
+    'wellbore_marker_frame'
```

Comment:
This list should be the actual public classes and methods, not the inner submodules